ftw/iframeblock/upgrades/20200305202319_ensure_correct_iframeresizer_js_order/jsregistry.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
```

Comment:
Good stuff, same comment as above over iframeResizer.contentWindow.min.js

Reply:
See the reply in the comment above.

Reply:
Ok, thanks